_includes/one-click-payments.md

```diff
@@ -138,7 +140,7 @@ Content-Type: application/json
 
 {
     "state": "Deleted",
-    "tokenType": "PaymentToken|RecurrenceToken",
+    "tokenType": "PaymentToken | RecurrenceToken",
```

Comment:
No pipes in examples, please. Explain the possible values in below property tables instead.

```suggestion
    "tokenType": "PaymentToken",
```